README.md

```diff
@@ -55,65 +55,13 @@ Register your application to use Microsoft Graph API using one of the following
 
-Refer devDependencies in [package.json](./package.json) for the compatible msal version and update that version in below.
-
-```html
-<script src="https://secure.aadcdn.microsoftonline-p.com/lib/<version>/js/msal.min.js"></script>
-```
-
-```typescript
-
-// Configuration options for MSAL @see https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/MSAL.js-1.0.0-api-release#configuration-options
-const msalConfig = {
-	auth: {
-		clientId: "your_client_id", // Client Id of the registered application
-		redirectUri: "your_redirect_uri",
-	},
-};
-const graphScopes = ["user.read", "mail.send"]; // An array of graph scopes
-
-// Important Note: This library implements loginPopup and acquireTokenPopup flow, remember this while initializing the msal
-// Initialize the MSAL @see https://github.com/AzureAD/microsoft-authentication-library-for-js#1-instantiate-the-useragentapplication
-const msalApplication = new Msal.UserAgentApplication(msalConfig);
-const options = new MicrosoftGraph.MSALAuthenticationProviderOptions(graphScopes);
-const authProvider = new MicrosoftGraph.ImplicitMSALAuthenticationProvider(msalApplication, options);
-```
-
-#### Creating an instance of ImplicitMSALAuthenticationProvider in node environment
-
-Refer devDependencies in [package.json](./package.json) for the compatible msal version and update that version in below.
-
-```cmd
-npm install msal@<version>
-```
-
-```typescript
-import { UserAgentApplication } from "msal";
-
-import { ImplicitMSALAuthenticationProvider } from "@microsoft/microsoft-graph-client/lib/src/ImplicitMSALAuthenticationProvider";
-import { MSALAuthenticationProviderOptions } from '@microsoft/microsoft-graph-client/lib/src/MSALAuthenticationProviderOptions';
-
-// An Optional options for initializing the MSAL @see https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/MSAL-basics#configuration-options
-const msalConfig = {
-	auth: {
-		clientId: "your_client_id", // Client Id of the registered application
-		redirectUri: "your_redirect_uri",
-	},
-};
-const graphScopes = ["user.read", "mail.send"]; // An array of graph scopes
-
-// Important Note: This library implements loginPopup and acquireTokenPopup flow, remember this while initializing the msal
-// Initialize the MSAL @see https://github.com/AzureAD/microsoft-authentication-library-for-js#1-instantiate-the-useragentapplication
-const msalApplication = new UserAgentApplication(msalConfig);
-const options = new MSALAuthenticationProviderOptions(graphScopes);
-const authProvider = new ImplicitMSALAuthenticationProvider(msalApplication, options);
-```
+    > **Important Note:** MSAL is supported only for frontend applications, for server-side authentication you have to implement your own AuthenticationProvider. Learn how you can create a [Custom Authentication Provider](./docs/CustomAuthenticationProvider.md).
```

Comment:
Can't they use [msal node](https://www.npmjs.com/package/@azure/msal-node)?

Reply:
No the authentication flows  are different.

Reply:
@ddyett @darrelmiller  would know better about this.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/13263
https://github.com/Azure/azure-sdk-for-js/issues/12565#issuecomment-732429468
My knowledge is that working with a single dependency that is @azure/identity which provides a lot of authentication flows (and plans to bring in more auth flows) would reduce work for us keeping the implementation short rather than taking separate dependencies or multiple implementations to support node or browser auth.

Reply:
ok. Thanks for looking into that further. I was confused about why they'd "mask" capabilities they get for "free" from MSAL that customers might need. But hopefully, in the fullness of time, this current PR adding azure identity adds support for a whole bunch of flows (browser, backend, etc...). Resolving